spec/views/user/user_show_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.feature 'User show page', type: :feature do
+  include Rails.application.routes.url_helpers
+  let!(:user) { User.create(name: 'Menen', photo: 'menen.jpg', postcounter: 5) }
+
+  before do
+    visit user_path(user)
+  end
+
+  it 'show user profile picture' do
+    expect(page).to have_css("img[src*='menen.jpg']")
+  end
+  it 'show the users username' do
+    expect(page).to have_content(user.name)
+  end
+
+  it 'showthe number of posts the user has written.' do
+    expect(page).to have_content("Number of posts : #{user.postcounter}")
+  end
+
+  it 'show the bio of the user' do
+    expect(page).to have_content(user.bio)
+  end
+
+  it 'shows the user first three posts' do
+    user.posts.limit(3).each do |post|
+      expect(page).to have_content(post.title)
+    end
+  end
```

Comment:
- You have done well so far, according to the project requirements, you are missing some required tests. Please fix this 🙏. The missing tests are:
* Tests that assert you can see a button that lets me view all of a user's posts.
* Tests that redirect to the post's show page when you click on a user's post.
* Tests that redirect to the user's post's index page when you click on a **see all posts**